great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -305,7 +305,7 @@ def __init__(  # noqa: C901, PLR0912, PLR0913, PLR0915
         connection_string: Optional[str] = None,
         url: Optional[str] = None,
         batch_data_dict: Optional[dict] = None,
-        create_temp_table: bool = True,
+        create_temp_table: bool = False,
```

Comment:
This shouldn't matter or need to change.
I'll look into this.

Probably also related to these issues
 1. https://github.com/great-expectations/great_expectations/issues/8147
 2. https://github.com/great-expectations/great_expectations/issues/8618
 3. https://github.com/great-expectations/great_expectations/issues/8325

Reply:
That seems to be the issue.
We should revert this change once this merges 👇 
- https://github.com/great-expectations/great_expectations/pull/9118